setup.py

```diff
@@ -1,94 +1,4 @@
-from os import path
-from setuptools import setup, find_packages
+from setuptools import setup
```

Comment:
Also, it looks like if we use [the newest version of `pip`](https://pip.pypa.io/en/stable/reference/build-system/pyproject-toml/#editable-installation) we will not need the `setup.py` workaround. We may still want to keep it though since some users will not be using the most recent version of `pip`.